docs/mp/graphql/01_mp_graphql.adoc

```diff
@@ -0,0 +1,223 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019, 2020 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= MicroProfile GraphQL
+:h1Prefix: MP
+:pagename: microprofile-graphql
+:description: Helidon GraphQL MicroProfile
+:keywords: helidon, graphql, microprofile, micro-profile
+
+
+The Microprofile GraphQL APIs are an extension to <<microprofile/01_introduction.adoc, Helidon MP>>
+to allow building of applications that can expose a GraphQL endpoint and optionally a link:https://github.com/graphql/graphiql[GraphiQL] API.
+
+== About the MicroProfile GraphQL Specification
+Helidon MP implements the MicroProfile GraphQL
+link:https://github.com/eclipse/microprofile-graphql[spec] version 1.0.3.
```

Comment:
Version 1.0.3 is the first version we were able to certify against. I can put an attribute in docs/sitegen.yaml which points to the version we are using in the pom.xml so this will be automatic and clear.